### DISCUSSION TITLE
rofi doesn't scroll with touchpad on Hyprland

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [davatorium/rofi#1987] [Q&A]
Hi,

I cannot scroll through the rofi elements with my touchpad by a two fingers swipe. It works on all other apps but not with rofi.

Need to use the scrollbar or the arrow keys instead.

On my desktop PC, I can use the scrollwheel.

Any idea or solution for this issue?

Thanks!!!

**Comment by DaveDavenport** [comment]
Do you run this rofi, or the wayland fork?

@lbonn

**Reply by mylinuxforwork** [reply]
I use the package with yay -S rofi-wayland

**Reply by DaveDavenport** [reply]
@lbonn   do you know the answer to  this one?

**Reply by lbonn** [reply]
@mylinuxforwork true, I can reproduce it. Please open an issue there https://github.com/lbonn/rofi/issues

**Reply by mylinuxforwork** [reply]
Done. Thank you.
